app/labor/email_logic.rb

```diff
@@ -28,13 +27,19 @@ def should_receive_email?
 
   def get_articles_to_send
     fresh_date = get_fresh_date
+
     articles = if user_has_followings?
+                 experience_level_rating = (@user.experience_level || 5)
+                 experience_level_rating_min = experience_level_rating - 3.6
+                 experience_level_rating_max = experience_level_rating + 3.6
+
                  @user.followed_articles.
                    where("published_at > ?", fresh_date).
                    where(published: true, email_digest_eligible: true).
                    where.not(user_id: @user.id).
                    where("score > ?", 12).
-                   where("experience_level_rating > ? AND experience_level_rating < ?", (@user.experience_level || 5) - 3.6, (@user.experience_level || 5) + 3.6).
+                   where("experience_level_rating > ? AND experience_level_rating < ?",
```

Comment:
ActiveRecord also supports ranges here:

```ruby
where(experience_level_rating: experience_level_rating_min .. experience_level_rating_max)
```

This will be turned into a `BETWEEN` in SQL:

```SQL
SELECT  1 AS one FROM "articles" WHERE "articles"."experience_level_rating" BETWEEN $1 AND $2 LIMIT $3
```

Note that these are not strictly equivalent, since `BETWEEN` is inclusive and you are using strict equality comparisons here, but that can easily be remedied by adjusting the values.

Reply:
Although it's better to look at I would leave that for a separate PR because it doesn't really improve the query plan PostgreSQL will use AFAIK and a separate optimization anyway :)

Reply:
Totally, I should have been clearer about this. I just saw it and left a comment since many people are not aware of this AR query option.